homeworks/kostiantyn.nosatskyi_NosatskyiK/PokemonProject/style.css

```diff
@@ -0,0 +1,28 @@
+h1 {
+  font-size: 48px ;
+  font-family: Roboto, serif ;
+  font-weight: 700 ;
+}
+
+.main_text {
+  font-family: Roboto, serif ;
+  font-size: 22px ;
+  font-weight: 400 ;
+}
+
+/* додав розміри для article щоб відповідати макету */
+
+article {
+  width: 575px ;
+  height: 479px ;
+}
+
+.link_MOC {
```

Comment:
Then you could skip this `MOC`, capital letters in CSS class names are not widely used.